### ISSUE TITLE
The download site has been banned in China

### ISSUE THREAD

**Issue by HuBowen1031** [EXALAB/AnLinux-App#356]
I am having trouble accessing your  website'raw.githubusercontent.com' to download the system.It appears to be banned.
The connection is refused.
Is there any way to fix it？

**Comment by jkoop** [comment]
[This](https://stackoverflow.com/a/62198387/3642588) may help

**Comment by xyzzklk** [comment]
I'm having the same problem too. So just use a VPN.

**Comment by HuBowen1031** [comment]
So,either I use the VPN,or modify the hosts file,right？Thanks for all of you.

**Comment by xyzzklk** [comment]
Yes. That's right. If you want to use a VPN on phone, try [SagerNet](https://github.com/SagerNet/SagerNet/releases/) or [Clash for Android](https://github.com/Kr328/ClashForAndroid/releases). Get free proxy servers from [ShadowShare](https://shadowshare.v2cross.com/).

**Comment by Nguyenhieu1a** [comment]
> [This](https://stackoverflow.com/a/62198387/3642588) or [this](https://www.tutorialexample.com/fix-raw-githubusercontent-com-is-not-accessed-in-china-and-download-files/) may help. I'm not in China; this is just a quick Google
